src/OgImage.js

```diff
@@ -15,6 +15,8 @@ import path from 'node:path';
 import url from 'node:url';
 import { sortObject } from './utils/index.js';
 
+const { File } = RenderPlugin;
```

Comment:
Instead of destructuring lets use `RenderPlugin.File` where currently `File` is used. I think that improves readability.